src/ResourceManager/Sql/Commands.Sql/ManagedDatabase/Cmdlet/NewAzureSqlManagedDatabase.cs

```diff
@@ -0,0 +1,185 @@
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.ResourceManager.Common.Tags;
+using Microsoft.Rest.Azure;
+using System.Management.Automation;
+using System.Collections;
+
+namespace Microsoft.Azure.Commands.Sql.ManagedDatabase.Cmdlet
+{
+    /// <summary>
+    /// Cmdlet to create a new Azure Sql Managed Database
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "AzureRmSqlManagedDatabase", SupportsShouldProcess = true)]
+    public class NewAzureSqlManagedDatabase : AzureSqlManagedDatabaseCmdletBase<AzureSqlManagedDatabaseModel>
+    {
+        protected const string CreateNewByNameAndResourceGroupParameterSet =
+            "CreateNewManagedDatabaseFromInputParameters";
+
+        protected const string CreateNewByInputObjectParameterSet =
+            "CreateNewManagedDatabaseFromAzureSqlManagedInstanceModelInstanceDefinition";
+
+        /// <summary>
+        /// Gets or sets the name of the managed database to create.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 0,
+            HelpMessage = "The name of the Azure SQL Managed Database to create.")]
+        [Alias("Name")]
+        [ValidateNotNullOrEmpty]
+        public string ManagedDatabaseName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the Azure Sql Managed instance to use
+        /// </summary>
+        [Parameter(ParameterSetName = CreateNewByNameAndResourceGroupParameterSet,
+            Mandatory = true,
+            Position = 1,
+            HelpMessage = "The Azure Sql Managed Instance name.")]
+        [ValidateNotNullOrEmpty]
+        public override string ManagedInstanceName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the resource group to use.
+        /// </summary>
+        [Parameter(ParameterSetName = CreateNewByNameAndResourceGroupParameterSet,
+            Mandatory = true,
+            Position = 2,
+            HelpMessage = "The name of the resource group.")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public override string ResourceGroupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the Azure SQL Managed Database collation to use
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The collation of the Azure SQL Managed Database collation to use.")]
+        [ValidateNotNullOrEmpty]
+        [PSArgumentCompleter("SQL_Latin1_General_CP1_CI_AS", "Latin1_General_100_CS_AS_SC")]
+        public string Collation { get; set; }
+
+        /// <summary>
+        /// Gets or sets the tags associated with the Azure Sql Managed Database
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The tags to associate with the Azure Sql Managed Database")]
+        [Alias("Tag")]
+        public Hashtable Tags { get; set; }
```

Comment:
All of these should be "Tag" with an alias to "Tags" - this will fix your build issue.